elasticsearch/Dockerfile

```diff
@@ -3,8 +3,8 @@ MAINTAINER qi.zheng@intel.com
 
 ARG swupd_args
 
-RUN swupd update $swupd_args \
-	&& swupd bundle-add elasticsearch $swupd_args \
+RUN swupd update --no-boot-update $swupd_args
+RUN swupd bundle-add elasticsearch $swupd_args \
 	&& rm -rf /var/lib/swupd/*
```

Comment:
We should remove the whole /var/lib/swupd directory as what's being done for the cgit Dockerfile